veil_falls_caves.logic

```diff
@@ -1,11 +1,89 @@
 area VeilFallsCaves:
-  room C1F1BehindBreakableWall
-  room C1F1BreakableWall
-  room C1F2Climb
-  room C1F3PushClones
-  room C1GFEntrance
-  room C2BF1WaterRupees
-  room C2F1
-  room C2GFEntry
-  room C2GFSideRoom
-  room WaterfallCave
+  room C1F1BehindBreakableWall:
+    node Main:
+      chest Chest
+      door Stair
+      door Door
+  room C1F1BreakableWall:
+    node Main:
+      door Stair
+      door South
+      enemy RedBubble
+    node BombWallExit:
+      exit BombWall
+    node BombWallEntrance
+      entrance BombWall
+    Main -> BombWallEntrance: ( item BombBag | flag BombWallBombed )
+    BombWallExit -> Main: ( item BombBag | flag BombWallBombed )
```

Comment:
This here can be done, but if the player has zero bombs they are softlocked, so do we remove this possibility?